fuzz/fuzz_targets/validation.rs

```diff
@@ -0,0 +1,69 @@
+#![no_main]
+use apollo_compiler::ApolloCompiler;
+use apollo_rs_fuzz::generate_valid_operation;
+use libfuzzer_sys::fuzz_target;
+use log::debug;
+use router_bridge;
+// use serde_json::json;
+// use serde_json::Value;
+use std::panic;
+
+fuzz_target!(|data: &[u8]| {
+    let (op, ts) = match generate_valid_operation(data) {
+        Ok(d) => (d.0, d.1),
+        Err(_err) => {
+            return;
+        }
+    };
+
+    let mut compiler = ApolloCompiler::new();
+    compiler.add_executable(&op, "op.graphql");
+    compiler.add_document(&ts, "ts.graphql");
+    let rust_diagnostics = compiler.validate();
+
+    debug!("======= OPERATION =======");
+    debug!("{}", op);
+    debug!("========================");
+    debug!("{}", ts);
+    debug!("========================");
+    let js_diagnostics =
+        router_bridge::validate::validate(&ts, &op).expect("js validation should succeed");
```

Comment:
```suggestion
        router_bridge::validate::validate(&ts, &op).expect("bridge should return validation result");
```
would emphasise here that it's not that JS validation did not pass, but that something else went wrong with Deno or somewhere, not the actual GraphQL validation